app/views/quotes/index.html.erb

```diff
@@ -0,0 +1,9 @@
+<h1>Search Results: <%=@search_term%> </h1>
+<p>Total quotes: <%= @results.num_results %></p>
+
+<% @results.quotes.each_with_index do |quote, index| %>
+  <div id="quote-<%= index %>">
```

Comment:
See my comment in your `feature spec`. You can use this id to test that the total count is 10.